mesonbuild/interpreter/interpreter.py

```diff
@@ -1087,6 +1087,11 @@ def func_get_option(self, nodes: mparser.BaseNode, args: T.Tuple[str, T.Optional
             value_object, value = self.coredata.optstore.get_option_from_meson_file(options.OptionKey(optname, self.subproject))
         except KeyError:
             fallback = kwargs.get('fallback', None)
+            if fallback is None and optname == 'b_vscrt':
```

Comment:
Is it really specific to `b_vscrt`, or shoud it be `optname.startswith('b_')` ?

Reply:
No it is not, but I want to keep the BW compatibility gap as small as possible. If nobody complains about other values, then we don't really need to support converting them and everything is simpler. If more are reported, then we should just do all `b_` ones. But not before.

Reply:
I frankly think this should be done for all options period, and just warn on all builtin options.

Reply:
Yes, that should be done for all b_ options, it was intentional to allow that. I'm not against a deprecation warning if we now add a fallback value to get_option().

Reply:
I replied elsewhere on why the fallback kwarg sounds good and useful in theory, but is an unnecessary complication  in practice. My vote is for keeping old behavior for builtin options, without any warning even.

Reply:
> My vote is for keeping old behavior for builtin options, without any warning even.

+1